src/com/rath/rathbot/action/ActionReport.java

```diff
@@ -0,0 +1,18 @@
+
+package com.rath.rathbot.action;
+
+import java.time.Instant;
+
+import sx.blah.discord.handle.obj.IUser;
+
+public class ActionReport extends RBDiscAction {
+  
+  public ActionReport(Instant timestamp, IUser issuedUser, IUser infringingUser) {
+    super(timestamp, issuedUser, infringingUser);
+  }
+  
+  @Override
+  public String getActionMessage() {
+    return this.timestamp + ": " + this.getIssuer().getName() + " reported " + this.infringingUser.getName() + ".";
+  }
+}
```

Comment:
Careful with not ending a file on a blank line. I don't think the Java compiler does, but some compilers will have a fit when you do this. It's good practice just to do it.